Sources/Controllers/2.0/OAGPXListViewController.mm

```diff
@@ -536,27 +534,27 @@ -(void)viewWillAppear:(BOOL)animated {
     [self applySafeAreaMargins];
 }
 
--(UIView *) getTopView
+- (UIView *) getTopView
```

Comment:
If you are using constraints (which you should), please remove this and three methods below that were used before the introduction of autolayout and are completely useless now.